app/AppKernel.php

```diff
@@ -21,6 +21,7 @@ public function registerBundles()
             new WhiteOctober\PagerfantaBundle\WhiteOctoberPagerfantaBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(), // used for initial population of non-SQLite databases in production envs
```

Comment:
Yes and no. Even a real application, when first deployed somewhere, may need e.g. an initial user inserted into the database ;)